common/templates/common/blocks/bubble_map_chart.html

```diff
@@ -1,4 +1,4 @@
-{% if value.title %}<h2>{{ value.title }}</h2>{% endif %}
+{% if static_graphic is None and value.title %}<h2>{{ value.title }}</h2>{% endif %}
```

Comment:
This hides the title if the graphic is a "static_graphic" when used in blog listing page or as primary image on the top of a page.